akka-docs/src/main/categories/additional-sink-and-source-converters.md

```diff
@@ -0,0 +1,21 @@
+Sources and sinks for integrating with `java.io.InputStream` and `java.io.OutputStream` can be found on
```

Comment:
where is this linked from?

Reply:
They're included in the index by https://github.com/akka/akka/pull/24928/files#diff-1b87779e2edf5ad483fd246e3a27e03dR181